shiny/Data-Ingest/app.R

```diff
@@ -58,30 +58,37 @@ ui <- dashboardPage(
 server <- function(input, output) {
   options(shiny.maxRequestSize=30*1024^2) #maximum file input size
   
-   #path <- PEcAn.utils::read_web_config(config.php)
+   path <- PEcAn.utils::read_web_config("../../web/config.php")
     
    d1d <- eventReactive(input$D1Button, { PEcAn.data.land::dataone_download(input$id, filepath = path) }) #run dataone_download with input from id on click
   
   output$identifier <- renderText({
     d1d()
   })
   
-   # output$debug <-
-  
- # 
-  
-  output$upload <- renderTable({
-    if(is.null(data())){return()}
-    input$file
-  })
```

Comment:
Is this chunk deleted because it was the wrong approach? Cause it was superseded by something else?